projects/Mallard/src/components/EditionsMenu/__tests__/__snapshots__/EditionsMenu.spec.tsx.snap

```diff
@@ -27,23 +28,12 @@ exports[`EditionsMenu should display a EditionsMenu with correct styling and alt
               "subTitle": "Weekend",
               "title": "Austraila",
             },
+            "locale": "en_AU",
             "notificationUTCOffset": 1,
             "subTitle": "Published every day by 9:30am (AEST)",
             "title": "Australia Daily",
             "topic": "au",
           },
-          Object {
```

Comment:
Is it possible to write snapshot test for dynamic list of items?

Reply:
Yes and no. You can mock out the dynamic list. So test it with static data separately, then the dynamic side is mocked in a higher level component

Reply:
I would say the regional editions are static in my opinion